ThePensionsRegulator.Frontend/wwwroot/tpr/tpr-search.js

```diff
@@ -0,0 +1,124 @@
+﻿import { Accordion } from '/govuk/all.min.js';
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
There are no tests for this JS. Our test coverage for other JS in this library is pretty good.

There are no tests for the C# either. That's true for all our components and not a problem I expect you to solve. FYI I want to find time to get our own version of conformance tests working - we have it for the task list so it should be similar, and then we could much more easily add tests.